docs/meshstack.api.md

```diff
@@ -0,0 +1,6 @@
+---
+id: meshstack.api
+title: API
+---
+
+meshStack provides APIs that can be consumed by third-party applications. This allows the integration of meshStack into established company processes provided by other applications. The API documentation can be found in every meshStack installation in the footer of meshPanel. It provides all details needed to use the API. You can also find the deployed API documentation [here](https://federation.meshcloud.io/docs/index.html).
```

Comment:
f: don't hardlink to meshcloud-prod, instead include notes about relative url ?

Reply:
hmm, in that case i like the the link, because it is currently the only option we have to provide this API to new customers, who are just checking our docs for functionality we provide. So i would keep this link for now. We can change it, when we include the API docs into this public doc